docker-compose.yml

```diff
@@ -21,11 +21,28 @@ services:
   # ------------------------------------------------------------------
   # After this point is dev-related services.
   # ------------------------------------------------------------------
-
-  # This is the base web-service.
-  web:
+  base:
     image: local/antenna_dev
     volumes:
       - .:/app
+    environment:
+      - ANTENNA_ENV=${ANTENNA_ENV}
```

Comment:
The one sucky thing about this is that it spits out a WARNING every time you don't specify it. Blech.